core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -275,6 +275,13 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
           )
         }
       }
+    } ~
+    path("admin" / "submissions" / "queueStatusByUser") {
```

Comment:
I wasn't quite sure about the best endpoint name -- if anyone has other suggestions, let me know.